shell/platform/darwin/ios/framework/Source/FlutterTextInputPlugin.mm

```diff
@@ -269,8 +366,54 @@ - (BOOL)isEqualTo:(FlutterTextRange*)other {
 }
 @end
 
+// A FlutterTextInputView that masquerades as a UITextField, and forwards
+// selectors it can't respond to to a shared UITextField instance.
+//
+// Relevant API docs claim that password autofill supports any custom view
+// that adopts the UITextInput protocol, automatic strong password seems to
+// currently only support UITextFields, and password saving only supports
+// UITextFields and UITextViews, as of iOS 13.5.
+@interface FlutterSecureTextInputView : FlutterTextInputView
+@property(nonatomic, strong, readonly) UITextField* textField;
+@end
+
+@implementation FlutterSecureTextInputView {
+  UITextField* _textField;
```

Comment:
nit: This isn't necessary since the property will declare it.

Reply:
I think this is required because the property is `readOnly` and I overrode its getter so it will not get synthesized?